### PR TITLE
feat:(#11) - Added common components for User Profile 

### PR DESCRIPTION
**Pull request by saviomartin**
## Part of #11

# Explain the feature/fix

Added common components that could be reused for User profile implementation and added some SVGs to be used on the profile page.

## Does this PR introduce a breaking change

No

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Comment by rejahrehim** [comment]
Resolve conflicts.

File: src/components/User/Common/Input.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Icon from "../../Common/Icon";
+
+const Input = ({ placeholder, icon }) => {
+    return (
+        <>
```
Comment (rejahrehim) [inline src/components/User/Common/Input.js:6]: ?

File: src/components/User/Common/Input.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                <span class="z-10 leading-snug font-normal absolute text-center text-gray-400 absolute bg-transparent rounded text-base items-center justify-center w-8 pl-3 py-3">
+                    <Icon name={icon} size="w-2" />
+                </span>
+                <input
+                    type="text"
+                    placeholder={placeholder}
+                    class="px-3 py-3 placeholder-gray-400 text-gray-700 relative bg-white bg-white rounded text-sm border border-gray-400 outline-none focus:outline-none focus:shadow-outline w-full pl-10"
+                />
+            </div>
+        </>
```
Comment (rejahrehim) [inline src/components/User/Common/Input.js:17]: ?

**Review by rejahrehim — CHANGES_REQUESTED** [review]

**Comment by riginoommen** [comment]
Can you please rebase this PR

**Review by riginoommen — APPROVED** [review]

**Review by rejahrehim — APPROVED** [review]

**Pull request merged**